source/org/zfin/framework/presentation/GONavigationMenu.java

```diff
@@ -17,7 +17,8 @@ public GONavigationMenu() {
             title(NavigationMenuOptions.SUMMARY).showCount(false),
             title(NavigationMenuOptions.RELATIONSHIPS).showCount(false),
             title(NavigationMenuOptions.EXPRESSION).showCount(false),
-            title(NavigationMenuOptions.PHENOTYPE).showCount(false)
+            title(NavigationMenuOptions.PHENOTYPE).showCount(false),
+            title(NavigationMenuOptions.CITATION).showCount(false)
```

Comment:
I remember that I specifically removed citations for AO term pages. Is that not true any longer?

Reply:
Oh, that's good to know. I thought this was a mistake. I think there are contradicting instructions in some of the tickets, for example, the testing instructions on ZFIN-8653 say:

"
Go to an AO term page, e.g.
action/ontology/prototype/ZDB-TERM-100331-27#summary
and make sure the reference section is names 'Citations' and not 'Citation'.
Check all other prototype / entity pages for this naming.
"

Reply:
Ceri would know the ultimate answer for this.

Reply:
I spoke with Yvonne about this and she made the call to keep the citations section. The Otic Vesicle is one example of a term that has a citation (https://zfin.org/action/ontology/prototype/ZDB-TERM-100331-49#citation)